src/Robot.elm

```diff
@@ -0,0 +1,89 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Browser.Events
+import Html exposing (Html, button, div, text)
+import Html.Attributes as Attributes exposing (style)
+import Html.Events exposing (onClick)
+import Json.Decode
+
+main =
+    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }
+
+
+type alias Model =
+    { xblue : Int, yblue : Int , xred : Int, yred : Int}
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
+    ( { xblue = 50, yblue = 50 ,xred = 10, yred = 10}, Cmd.none )

```

Comment:
Simplify `init` by removing unnecessary "glue" outside - it shouldn't take a parameter if it's ignored, and shouldn't return a `Cmd Msg` if it's just `Cmd.none`:
```elm
init : Model
init =
    { x = 50
    , y = 50
    }
```
and then in `main`, use `always`:
```elm
main : Program () Model Msg
main =
    Browser.element
        { init = always (init, Cmd.none)
        ...
```